### DISCUSSION TITLE
How to Yank the Path of the Current File?

### DISCUSSION THREAD

**Discussion by JuneKelly** [LazyVim/LazyVim#5076] [Q&A] [answered]
Hey folks 👋  I've been running `let % = expand("%")` to copy the (relative) path of the current file to the clipboard. But I wonder if there's an existing keybinding I could use to do the same thing?

**Comment by dpetka2001** [comment, marked as answer]
Nope. You would have to create your own. I have the following to get the full filename+linenumber
```lua
vim.keymap.set("n", "<leader>y", function()
  vim.fn.setreg(vim.v.register, vim.fn.expand("%:p") .. ":" .. vim.fn.line("."))
end, { desc = "Copy filename+line to clipboard" })
```
Customize it to your liking to get what you want.
